### PR TITLE
Add information on using pre-built / contributed dashboards.

### PR DESCRIPTION
**Pull request by tim-seoss**
The docs don't currently mention this possibility, leaving new users to either assume this doesn't exist or begin googling on how to do this...

### CODE REVIEW

File: content/docs/visualization/grafana.md [hunk 1]
```diff
@@ -61,3 +61,8 @@ legend format string `{{method}} - {{status}}`.
 
 The following shows an example Prometheus graph configuration:
 [![Prometheus graph creation](/assets/grafana_qps_graph.png)](/assets/grafana_qps_graph.png)
+
+### Importing pre-built dashboards from Grafana.net
+Grafana.net maintains [a collection of shared dashboards](https://grafana.net/dashboards) which can be downloaded and used with standalone instances of Grafana.  Use the Grafana.net "Filter" option to browse 'Prometheus' specific dashboards only.
```
Comment (beorn7) [inline content/docs/visualization/grafana.md]: How about:
... to browse dashboards for the "Prometheus" data source only.

**Comment by beorn7** [comment]
Some spelling nits commented upon.

@grobie as our Grafana expert, please check the content.

**Comment by tim-seoss** [comment]
Reworked based on suggestions.  Partially rewritten for clarity (hopefully).

File: content/docs/visualization/grafana.md [hunk 2]
```diff
@@ -61,3 +61,8 @@ legend format string `{{method}} - {{status}}`.
 
 The following shows an example Prometheus graph configuration:
 [![Prometheus graph creation](/assets/grafana_qps_graph.png)](/assets/grafana_qps_graph.png)
+
+### Importing pre-built dashboards from Grafana.net
+Grafana.net maintains [a collection of shared dashboards](https://grafana.net/dashboards) which can be downloaded and used with standalone instances of Grafana.  Use the Grafana.net "Filter" option to browse dashboards for the "Prometheus" datasource only.
+
+You must currently manually edit the downloaded JSON files and correct the "datasource:" entries to reflect the Grafana data source name which you chose for your Prometheus server.  Use the "Dashboards" → "Home" → "Import" option to import the edited dashboard file into your Grafana install.
```
Comment (juliusv) [inline content/docs/visualization/grafana.md]: For quoting "datasource:", backticks make more sense, as it's code and not UI.

File: content/docs/visualization/grafana.md [hunk 3]
```diff
@@ -61,3 +61,8 @@ legend format string `{{method}} - {{status}}`.
 
 The following shows an example Prometheus graph configuration:
 [![Prometheus graph creation](/assets/grafana_qps_graph.png)](/assets/grafana_qps_graph.png)
+
+### Importing pre-built dashboards from Grafana.net
+Grafana.net maintains [a collection of shared dashboards](https://grafana.net/dashboards) which can be downloaded and used with standalone instances of Grafana.  Use the Grafana.net "Filter" option to browse dashboards for the "Prometheus" datasource only.
```
Comment (juliusv) [inline content/docs/visualization/grafana.md]: Could you wrap the lines similarly to the rest of the file?

**Comment by juliusv** [comment]
:+1: besides last nits. Thanks!

**Comment by juliusv** [comment]
Thanks!

**Pull request merged**